src/CMakeLists.txt

```diff
@@ -296,6 +296,20 @@ add_subdirectory(json_spirit)
 include_directories(SYSTEM "${CMAKE_SOURCE_DIR}/src/xxHash")
 include_directories(SYSTEM "${CMAKE_SOURCE_DIR}/src/rapidjson/include")
 
+if(WITH_RADOSGW AND WITH_RADOSGW_AWS_ENDPOINT)
+  set(BUILD_ONLY "lambda" "sns")
```

Comment:
what is this variable for?

Reply:
it is used inside the aws-sdk submodule cmake - indicating it to build only sns and lambda libraries.
we set it from here instead of changing it in the submodule.

Reply:
you are referencing `https://github.com/prateek2211/aws-sdk-cpp`, right? @prateek2211 could you update `aws-sdk-cpp` to ensure these variables (i assume they are cache entries) are prefixed with something like `AWS_SDK_` ? to avoid name conflicting and for better readability? the name of `BUILD_ONLY` is way too general.

Reply:
ok

Reply:
if we are changing the submodule cmake anyway, we can just set it there?